en/firefox/browsers/compare/ie.ftl

```diff
@@ -0,0 +1,55 @@
+
+# "sunset" here means Microsoft is officially ending production of Internet Explorer. Alternative: "retire"
+compare-ie-as-microsoft-has-made-the-move = As { -brand-name-microsoft } has made the move to sunset the { -brand-name-ie } browser, it no longer supports any version for { -brand-name-ios }, and has never been available for { -brand-name-android }. Which means unless you’re running a Windows-based laptop or desktop, you won’t have access to your bookmarks, browsing history, saved passwords, and other information that modern browsers sync across devices.
+compare-ie-firefox-works-on-any-platform = { -brand-name-firefox } works on any platform, including { -brand-name-windows }, { -brand-name-mac }, { -brand-name-linux }, { -brand-name-android } and { -brand-name-ios }. Which also means you can sync all your information across platforms. So if you’re browsing on a { -brand-name-windows }-based laptop, you can pick up where you left off when you move to browsing on your { -brand-name-iphone } or { -brand-name-android } device. This convenience should come standard with any modern web browser, and is sorely lacking with { -brand-name-ie }.
+compare-ie-there-was-a-time-not-so-long = There was a time not so long ago where { -brand-name-ie } was the most popular and widely used browser in the world. Times have changed and so has technology, but unfortunately { -brand-name-ie } has pretty much stayed the same. { -brand-name-microsoft } itself openly implores users to stop using { -brand-name-ie } and instead switch to their newer Chromium-based { -brand-name-edge } browser.
+
+# Variables:
+#   $attrs (string) - link to /firefox/download/thanks/ with additional attributes for analytics
+# "Nana" is a common nickname for a grandmother in many English-speaking countries. Use a similar name for a grandmother in your language or culture.
+compare-ie-our-opinion-is-just-to-go-with = Our opinion is just to go with a trusted, private browser with a track record of delivering a great experience across devices. In a head-to-head comparison, it’s really no contest at all. { -brand-name-firefox } is hands down the winner across all assessment categories. If you do find yourself at Nana’s house firing up { -brand-name-ie }, maybe you want to do Nana a favor and <a { $attrs }>download { -brand-name-firefox }</a> for her.
```

Comment:
Maybe we need to add a comment for `firing up` as in `Nana's house firing up IE...`